pkg/chartutil/requirements_test.go

```diff
@@ -33,3 +36,173 @@ func TestLoadRequirementsLock(t *testing.T) {
 	}
 	verifyRequirementsLock(t, c)
 }
+func TestRequirementsTagsNonValue(t *testing.T) {
+	c, err := Load("testdata/subpop")
+	if err != nil {
+		t.Fatalf("Failed to load testdata: %s", err)
+	}
+	// tags with no effect
+	v := &chart.Config{Raw: string("tags:\n  nothinguseful: false\n\n")}
```

Comment:
Oh man there is no telling at what small hour I did that :) and then copied it 11 times.